src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+
+<body>
+
+    <section class="container">
+        <h1>Leaderboard</h1>
+        <div class="container2">
+            <div class="recentDiv">
+                <div class='recent'>
+                    <h2>Recent score</h2>
+                    <button>Refresh</button>
+                </div>
+                <div id="scoreDiv">
+                    <ul id="ul"></ul>
+                </div>
+            </div>
+            <div class="score">
+                <h2>Add your score</h2>
+                <input type="text" id="nameInput" placeholder="Your name">
+                <input type="text" id="scoreInput" placeholder="Your score">
+                <button id="submitBtn" type="submit">Submit</button>
```

Comment:
| @pushkar1995 Form | Wireframe Form |
|-|-|
| ![image](https://github.com/pushkar1995/leaderboard/assets/93039359/ba5da283-986d-4144-a045-329b0e941292) | ![image](https://github.com/pushkar1995/leaderboard/assets/93039359/5a0235b3-8858-47e9-b2d9-fd164373bca0) |

- [ ] It would be nice if you follow the project wireframe then you can see that the submit button is on the left side of the form 🤔 So kindly put your button on the left side too to make your project follow the given wireframe 👍 to make your project more professional 😊